components/Text/text.css

```diff
@@ -0,0 +1,65 @@
+@import url('/utils.css');
+.heading-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.h1{
+    font-size: 3.5rem;
+}
+.h2{
+    font-size: 3rem;
+}
+.h3{
+    font-size: 2.5rem;
+}
+.h4{
+    font-size: 2rem;    
+}
+.h5{
+    font-size: 1.75rem;
+}
+.h6{
+    font-size: 1.25rem;
+}
+
+.xs-txt{
+    font-size: 1rem;
+}
+.sm-txt{
+    font-size: 1.5rem;
+}
+.md-txt{
+    font-size: 2rem;
+}
+.lg-txt{
+    font-size: 3rem;
+}
+.xl-txt{
+    font-size: 4rem;
+}
+.grey-txt{
+    color: var(--clr-grey);
+}
+.txt-center{
+    text-align: center;
+}
+.italic-txt{
+    font-style: italic;
+}
+.strike-txt{
+    text-decoration: line-through;
+}
+.underline-txt{
+    text-decoration: underline;
+}
+.bold{
```

Comment:
you could also add text prefix for these classes

Reply:
can you explain about text-prefix?